integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonDescribeFormattedCommand.scala

```diff
@@ -53,6 +53,21 @@ private[sql] case class CarbonDescribeFormattedCommand(
       (field.name, field.dataType.simpleString, colComment)
     }
 
+    /* Append non-schema columns */
+    val columns = relation.carbonTable.getTableInfo.getFactTable.getListOfColumns.asScala
+    val implicitColumns = for (column <- columns if column.getSchemaOrdinal == -1) yield {
+      (column.getColumnName, column.getDataType.getName.toLowerCase, "")
+    }
+
+    if (implicitColumns.nonEmpty) {
+      results ++= Seq(
+        ("", "", ""),
+        ("## Non-Schema Columns", "", "")
+      )
+
```

Comment:
remove empty lines

Reply:
Ok. Removed blank lines in the complete PR.